nodebuilder/p2p/pubsub.go

```diff
@@ -109,12 +115,21 @@ func topicScoreParams(network Network) map[string]*pubsub.TopicScoreParams {
 	return mp
 }
 
-func peerScoreParams(isBootstrapper bool, bootstrappers Bootstrappers) *pubsub.PeerScoreParams {
+func peerScoreParams(isBootstrapper bool, bootstrappers Bootstrappers, cfg Config) (*pubsub.PeerScoreParams, error) {
 	bootstrapperSet := map[peer.ID]struct{}{}
 	for _, b := range bootstrappers {
 		bootstrapperSet[b.ID] = struct{}{}
 	}
 
+	var ipColocFactWl []*net.IPNet //nolint
+	for _, strIP := range cfg.IPColocationFactorWhitelist {
```

Comment:
Applying rename suggestion over GH does not change it where it's used so this broken now